presto-hive/src/main/java/com/facebook/presto/hive/HiveUtil.java

```diff
@@ -279,6 +310,21 @@ static boolean isSplittable(InputFormat<?, ?> inputFormat, FileSystem fileSystem
         }
     }
 
+    public static boolean isLzoCompressedFile(Path filePath)
```

Comment:
Unless we plan to contribute this code back, please move these methods to a twitter-specific class for merge-friendliness.